rainbow-me/rainbowkit
# feat: add desktop download URL and instructions

- add a downloadUrls.desktop property
- add a wallet.desktop.instructions property
- set these on the Ledger connector to present desktop and mobile app download buttons for Ledger Live

demo here: https://645e04523bc76170fcab5872--ledger-rainbowkit-demo.netlify.app

testing:

- choose the Ledger Live wallet
- refuse to open Ledger Live when prompted
- press "GET"
- there should be app install buttons for desktop and mobile
- after pressing "Get the desktop app" it should show the instructions
- pressing "Connect" should take you back to the QR code screen and prompt you to open Ledger Live Desktop

![image](https://github.com/rainbow-me/rainbowkit/assets/106091551/dace0fc8-b28e-448a-9637-72c0109a3af3)
![image](https://github.com/rainbow-me/rainbowkit/assets/106091551/6944fde3-ec74-430b-b885-f578e61b9983)
